policy/release/cve.rego

```diff
@@ -203,7 +203,7 @@ _unpatched_vulnerabilities_deprecated := vulnerabilities if {
 
 _result_name := "SCAN_OUTPUT"
 
-_deprecated_result_name := "CLAIR_SCAN_RESULT"
+_deprecated_result_name := "SCAN_OUTPUT"
```

Comment:
We don't want this change, that is the name of the deprecated result and should stay as is.

Reply:
reverted